package.json

```diff
@@ -33,12 +33,13 @@
   "homepage": "https://github.com/atassis/run-script-webpack-plugin#readme",
   "devDependencies": {
     "@types/node": "^14.14.10",
-    "@types/webpack": "^4.41.25",
+    "webpack": "5.*.*",
```

Comment:
wildcards? why?

Reply:
Changed it to "5" simply, according to https://docs.npmjs.com/about-semantic-versioning it has the same effect. If you have a preference, let me know.

Reply:
I prefer to clearly set the version with caret sign `^` so it'ld gonna show the last version tested

Reply:
Okay, done